team.html

```diff
@@ -1,333 +1,472 @@
-        <p>Members of the CC are representatives of the main developers groups and stakeholders and are working by consensus.</p>
-        <p>The role of the CC is the overall coordination and management of the gammapy project, including:</p>
+        <p>
+            Current CC members (alphabetical order):
+        </p>
         <ul>
-          <li>Keeping a large-scale view of the Gammapy project (e.g. promotion, development strategies and priorities, publications and communications),</li>
-          <li>Making the link with astrophysics missions, especially CTA,</li>
-          <li>Taking care of funding application and manpower ressources,</li>
-          <li>Guarantee the project communication infrastructures (e.g. mailing list, github, ..),</li>
-          <li>Make decisions about Proposals for the Improvement of Gammapy (PIG).</li>
+            <li><strong>Bruno Khelifi</strong> (APC, France)</li>
+            <li><strong>Christoph Deil</strong> (MPIK, Germany)</li>
+            <li><strong>Régis Terrier</strong> (APC, France)</li>
+            <li><strong>Roberta Zanin</strong> (MPIK, Germany)</li>
         </ul>
-        The composition of the CC is:
+
+        <!-- See https://www.finance-ni.gov.uk/articles/roles-and-responsibilities-project-manager-->
+        <h2 class="project-manager">Project manager</h2>
+        <p class="lead">
+            The project manager is the non-technical executive lead for the Gammapy project.
+        </p>
+        <p>
+            The project manager is appointed by the Gammapy coordination committee, and works closely with the Gammapy
+            coordination committee, core developers and users.
+        </p>
+        <p>
+            Responsibilities include:
+        </p>
         <ul>
-          <li>Christoph Deil (MPIK, Germany)</li>
-          <li>Régis Terrier (APC, France)</li>
-          <li>Roberta Zanin (MPIK, Germany)</li>
-          <li>Bruno Khelifi (APC, France)</li>
+            <li>
+                Organise and drive all non-technical aspects of the project on a day-to-day basis.
+                Keep the overview of ongoing activities, schedules and action items and follow up
+                to make sure all important things get done.
+            </li>
+            <li>
+                Serve as Gammapy coordination committee secretary (schedule and moderate calls; give status reports;
+                write minutes)
+            </li>
+            <li>Organise all Gammapy calls and face-to-face meetings (for users and / or developers) via <a
+                    href="https://github.com/gammapy/gammapy-meetings/">gammapy-meetings</a>
+            </li>
+            <li>
+                Keep an overview and help coordinate all activities that have some involvement of Gammapy,
```

Comment:
all activities that have some involvement of Gammapy-> seems a bit too broad.
In the long run you don't want to follow every paper using gammapy, no?
maybe Keep an overview and coordinate general communication and teaching activities such as gammapy papers, presentations and tutorials.

Reply:
I agree with Regis. gammapy aims to be used by many people, e.g. people that are not developers, and our care is to be sure that gammapy is well acknowledge. We do not want to coordinate papers whose the results are made with gammapy, but we will coordinate the papers about gammapy itself!